### PR TITLE
Bump compat for MLJBase to 1 (keep existing compat)

### PR DESCRIPTION
**Pull request by rikhuijzer**

### CODE REVIEW

**Comment by rikhuijzer** [comment]
@ablaom can you merge this PR and release if you agree with this? I have no merge rights on this repository.

**Review by ablaom — APPROVED** [review]
Yes this should be good to go, thanks.

**Pull request merged**
